OrbitCore/SamplingProfiler.cpp

```diff
@@ -436,6 +435,10 @@ void SamplingProfiler::UpdateAddressInfo(uint64_t address) {
     function_address = address;
   }
 
+  if (address_info != nullptr) {
```

Comment:
This now actually seems to be functionally different. Before you only updated address_info->function_name when function was not a nullptr. I don't know what is correct.

Reply:
the only difference is that empty names are going to be updated with "???". I think this is fine.